orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/FindImageFromClusterTask.groovy

```diff
@@ -324,6 +324,14 @@ class FindImageFromClusterTask extends AbstractCloudProviderAwareTask implements
             mkDeploymentDetail((String) image.imageName, (String) image.amis[location.value][0], deploymentDetailTemplate, config)
           ]
         }
+        //Docker registry to deployment detail conversion
```

Comment:
should this be scoped to `titus`? Just making sure it's safe for other providers... (i think so, but not 100% sure)

Reply:
I didn't really want to add any specific cloud provider logic here.  This wasn't working with any of the docker centric cloud providers before anyways, so I feel like if someone wants to make this work with k8s they can follow this pattern.